admin/tools/docker/deployment/in2p3/env.example.sh

```diff
@@ -0,0 +1,20 @@
+
+# Set nodes names
```

Comment:
It would be nice to have some documentation on which variables to change in which scripts, and which ones to use depending on the use case.